jobs/config.json

```diff
@@ -3354,6 +3354,21 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-gce-pod-preemption-beta": {
```

Comment:
maybe remove "-beta" prefix? is it needed?

Reply:
It was convention that existing jobs had(for ex: gpu), so just followed it.

Reply:
ci/k8s-beta points to latest active release branch, maybe you want to test against head of master?

Reply:
So, I thought active release branch maps to 1.11 which we wanted to test against. Am I wrong in assuming so?

Reply:
normally new job starts with testing against latest of master, but if testing against 1.11 is WAI then this is fine

Reply:
Thanks. To be clear, my question is -  Say if we get started on 1.12 development, won't the release branch map 1.12?

Reply:
not until 1.12 branch is cut, which is way later than the start of 1.12 development

Reply:
Ohh now I understood why you were suggesting to make it master. I will change the job. Thanks for the clarification.